sentry/src/main/java/io/sentry/NoOpTransaction.java

```diff
@@ -10,31 +10,26 @@
 
 public final class NoOpTransaction implements ITransaction {
 
-  @Override
-  public void setName(@NotNull String name) {}
+  private static final NoOpTransaction instance = new NoOpTransaction();
 
-  @Override
-  public @NotNull Span startChild() {
-    return new Span(SentryId.EMPTY_ID, SpanId.EMPTY_ID, this, NoOpHub.getInstance());
+  private NoOpTransaction() {}
+
+  public static NoOpTransaction getInstance() {
+    return instance;
   }
 
   @Override
-  public @NotNull Span startChild(
-      final @NotNull String operation, final @NotNull String description) {
-    return startChild();
-  }
+  public void setName(@NotNull String name) {}
 
   @Override
-  public @NotNull Span startChild(final @NotNull SpanId parentSpanId) {
-    return startChild();
+  public @NotNull ISpan startChild(final @NotNull String operation) {
+    return NoOpSpan.getInstance();
   }
 
   @Override
-  public @NotNull Span startChild(
-      final @NotNull SpanId parentSpanId,
-      final @NotNull String operation,
-      final @NotNull String description) {
-    return startChild();
+  public @NotNull ISpan startChild(
+      final @NotNull String operation, final @Nullable String description) {
+    return startChild(operation);
```

Comment:
just a nit: you could return `NoOpSpan.getInstance();` directly so you spare a method call